examples/birefringence-and-phase.yml

```diff
@@ -0,0 +1,30 @@
+input_channel_names:
+- State0
+- State1
+- State2
+- State3
+reconstruction_dimension: 3
+birefringence:
+  transfer_function:
+    swing: 0.1
+  apply_inverse:
+    wavelength_illumination: 0.532
+    background_path: ''
+    remove_estimated_background: false
+    orientation_flip: false
```

Comment:
not sure if I understand this parameter. Is the orientation flip doing a mirror or a 180 deg rotation?

Reply:
I realized these are not implemented yet so we can change the naming afterwards